cp1616/src/cp1616_io_controller.cpp

```diff
@@ -66,6 +83,15 @@ Cp1616IOController::~Cp1616IOController()
   
 }
 
+void operator >> (const YAML::Node &node, ControllerModuleData &module)
+{
+  node["id"] >> module.id;
```

Comment:
This happens: 
`terminate called after throwing an instance of 'YAML::TypedKeyNotFound<std::string>'
what():  yaml-cpp: error at line 1, column 3: key not found: id`

Should I add some testing of data validity ?